lib/pipekit/repository.rb

```diff
@@ -36,7 +39,7 @@ def where(options)
     #
     # Returns a Hash or nil if none found.
     def find_by(options)
-      where(options).first
+      where(options, true).first
```

Comment:
So this is the bit that may be dangerous - we should check that pipedrive is not automatically making  fuzzy searches.